gym_roboy/envs/tests/test_robot_state.py

```diff
@@ -48,3 +48,13 @@ def test_robot_normalize_max_state():
     normed_min_state = MSJ_ROBOT.normalize_state(state=MSJ_ROBOT.new_min_state())
     assert np.allclose(normed_min_state.joint_angle, -max_angles)
     assert np.allclose(normed_min_state.joint_vel, -max_vels)
+
+
+def test_normalize_between_1_and_minus1():
+    high = np.random.random()
+    low = high - np.abs(np.random.random())
+
+    high_norm = MSJ_ROBOT._normalize_between_minus1_and1(high, max_val=high, min_val=low)
```

Comment:
Isn't normalization usually in the range of 0-1?

Reply:
There is different types of normalization and scalings. This particual is between -1 and 1 for symmetry, nothing more.